roles/php/tasks/main.yml

```diff
@@ -30,9 +31,15 @@
       - php72-php-soap
       - php72-php-xml
       - php72-php-gd
-      - composer
     state: present
 
+- name: Remove all php56 packages and composer
+  yum: 
```

Comment:
I put removal at the end so the install is not needlessly broken while executing the task. Remove 5.6 when 7.2 is fully configured

Reply:
I've added a commit to move it to the end